app/screens/phone-auth-screen/phone-input.tsx

```diff
@@ -210,17 +276,8 @@ export const PhoneInputScreen: React.FC = () => {
         )}
 
         <View style={styles.buttonsContainer}>
-          <GaloySecondaryButton
-            title={LL.PhoneInputScreen.whatsapp()}
-            containerStyle={styles.whatsAppButton}
-            loading={captchaLoading && messagingChannel === MessagingChannel.Whatsapp}
-            onPress={() => submitPhoneNumber(MessagingChannel.Whatsapp)}
-          />
-          <GaloyPrimaryButton
-            title={LL.PhoneInputScreen.sms()}
-            loading={captchaLoading && messagingChannel === MessagingChannel.Sms}
-            onPress={() => submitPhoneNumber(MessagingChannel.Sms)}
-          />
+          {secondaryButton}
```

Comment:
```suggestion
          {SecondaryButton}
```

component are by convention with an Uppercase afaik